.github/workflows/build-linux-cross.yml

```diff
@@ -19,12 +19,15 @@ jobs:
           sudo apt-get install -y --no-install-recommends \
                   build-essential \
                   gcc-14-riscv64-linux-gnu \
-                  g++-14-riscv64-linux-gnu
+                  g++-14-riscv64-linux-gnu \
+                  libcurl4-openssl-dev
```

Comment:
Does this work?
```suggestion
                  libcurl4-openssl-dev:riscv64
```

Reply:
Thanks, it works. I tested on my fork: https://github.com/ngxson/llama.cpp/actions/runs/14295363304/job/40061820312